performance-storage-service/pss_project/api/serializers/rest/metrics/MemoryMetricsSerializer.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework.serializers import Serializer, DecimalField
+from pss_project.api.models.rest.metrics.MemoryMetrics import MemoryMetrics, MemoryItemSummary
+
+
+class MemoryItemSummarySerializer(Serializer):
+    avg = DecimalField(max_digits=10, decimal_places=4, coerce_to_string=False)
+
+    def create(self, validated_data):
+        return MemoryItemSummarySerializer(**validated_data)
+
+
+class MemoryMetricsSerializer(Serializer):
+    rss = DecimalField(max_digits=10, decimal_places=4, coerce_to_string=False)
```

Comment:
I thought the `rss` and `vms` were integers. Is that not the case?

Reply:
Ah you are right! I will change them to `int` instead.

Reply:
They are `IntegerField` now